demisto_sdk/commands/content_graph/objects/integration_script.py

```diff
@@ -53,3 +58,33 @@ def get_supported_native_images(
                 native_image_config=file_to_native_image_config(),
             ).get_supported_native_image_versions(get_raw_version=True)
         return []
+
+    def get_python_version(self) -> Optional[Version]:
+        """
+        Get the python version from the script/integration docker-image in case it's a python image
+        """
+        if "python" not in self.type or not self.docker_image:
```

Comment:
are we sure all python integrations their docker image name contains python?

Reply:
yes